scripts/iql-lesson-sync/README.md

```diff
@@ -0,0 +1,57 @@
+# Upload to IBM Quantum Learning
```

Comment:
Nit: can we call the folder `learning-lesson-sync`? IQL is a jargon-y acronym.

Reply:
Fine with me